make.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
```

Comment:
for future scripts, recommend something more flexible and readable like typescript. example:

https://github.com/temporalio/samples-typescript/blob/main/.scripts/copy-shared-files.mjs 

but with: https://google.github.io/zx/typescript#typescript